app/controllers/users_controller.rb

```diff
@@ -32,4 +60,20 @@ def load_user
     flash[:warning] = t "users.user_not_found"
     redirect_to root_path
   end
+
+  def logged_in_user
+    unless logged_in?
```

Comment:
return unless logged_in?
store_location
flash[:danger] = t "users.login_remind"
redirect_to login_path

Reply:
Sorry to say that: After i logged in the example user, error occurred when i call user index, profile and settings. Is it ok for me to keep original one?